apps/integration/integration/ingests/index.py

```diff
@@ -25,6 +26,13 @@ def ingested_index(opensearch_client, container_handles, ingest_profile):
         index_ctx = HttpCrawlerIndex(
             profile="sort-all", opensearch=opensearch_client, importer=container_handles["importer"]
         )
+    elif ingest_profile == "jupyter-default-prep":
+        index_ctx = JupyterIndex(
+            nb_name="default-prep-script.ipynb", opensearch=opensearch_client, jupyter=container_handles["jupyter"]
+        )
+    elif ingest_profile == "jupyter-dev-example":
+        index_ctx = JupyterIndex(
+            nb_name="jupyter_dev_example.ipynb", opensearch=opensearch_client, jupyter=container_handles["jupyter"]
+        )
```

Comment:
Could this stuff be data-driven or based on patterns?  A bunch of if-then-else logic isn't ideal when the code is so similar.

Reply:
Could do something like that, sure (although I think python's `match` statement is only a thing in 3.12, and you really want that for pattern matching stuff... pretty sure until 3.12 if elif is how you do that). I'm not sure we're ever going to have enough ingest profiles that it's worth doing anything more complicated than simply enumerating them. remember, each one of these is a full sycamore ingest job + 48 opensearch queries, so the cost of adding to this section is not in the complexity/duplicativeness of this section